backend/capellacollab/tools/models.py

```diff
@@ -4,17 +4,40 @@
 
 from __future__ import annotations
 
-import typing as t
-
 import pydantic
 import sqlalchemy as sa
 from sqlalchemy import orm
 
 from capellacollab.core import database
-from capellacollab.tools.integrations import models as integrations_models
+from capellacollab.core.database import decorator
+
+DOCKER_IMAGE_PATTERN = r"^[a-zA-Z0-9][a-zA-Z0-9_\-/.:$]*$"
+
 
-if t.TYPE_CHECKING:
-    from .integrations.models import DatabaseToolIntegrations
+class ToolIntegrations(pydantic.BaseModel):
+    model_config = pydantic.ConfigDict(from_attributes=True, extra="forbid")
+
+    t4c: bool = pydantic.Field(
+        default=False,
+        description=(
+            "Enables support for TeamForCapella."
+            "If enabled, TeamForCapella repositories will be shown as model sources for corresponding models."
+            "Also, session tokens are created for corresponding sessions."
+            "Please refer to the documentation for more details."
```

Comment:
```suggestion
            "Enables support for TeamForCapella."
            " If enabled, TeamForCapella repositories will be shown as model sources for corresponding models."
            " Also, session tokens are created for corresponding sessions."
            " Please refer to the documentation for more details."
```

Use whitespaces after a dot (strings will automatically be concatenated.

Alternatively, use something like:

```suggestion
            textwrap.dedent("""\
            Enables support for TeamForCapella.
            If enabled, TeamForCapella repositories will be shown as model sources for corresponding models.
            Also, session tokens are created for corresponding sessions.
            Please refer to the documentation for more details.
            """
```

Reply:
Changed to whitespaces after the dot. Looks cleaner.